### PR TITLE
Mano screen invite notif e2e fix

### PR DESCRIPTION
**Pull request by xmano**
Fixes E2E test fail : 'Organiser logs in and verifies the accepted invitations'

### CODE REVIEW

File: apps/festival/festival-e2e/src/integration/dashboard/screening-invitations-2688.spec.ts [hunk 1]
```diff
@@ -20,13 +20,16 @@ const OrgName = orgsFixture.getByID(TestEVENT.org.id).denomination.public;
 const users  =  [ userFixture.getByUID(TestEVENT.by.uid) ];
 users.push(...invitedUsers.map(uid => userFixture.getByUID(uid)));
 users.push(userFixture.getByUID(USER.Ivo));
+//Admin
```
Comment (phpgeekfr) [inline apps/festival/festival-e2e/src/integration/dashboard/screening-invitations-2688.spec.ts:23]: is `USER.Daphney` an admin?

File: env/env.blockframes-mano.ts [hunk 2]
```diff
@@ -129,11 +129,15 @@ export const bigQueryAnalyticsTable = 'blockframes-staging.analytics_194475853.e
...
+export const centralOrgID = 'jnbHKBP5YLvRQGcyQ8In';
 
 // Import / Export parameters
 // =======
 
 export const backupBucket = 'mano-backups';
 export const heavyChunkSize = 2;
 export const chunkSize = 15;
+
+export function firebase(app?: string) {
```
Comment (phpgeekfr) [inline env/env.blockframes-mano.ts:145]: can you move this after `firebaseConfig` like other envs ?

**Review by phpgeekfr — APPROVED** [review]

File: apps/festival/festival-e2e/src/integration/dashboard/screening-invitations-2688.spec.ts (hunk 1 above)
Comment (xmano) [reply apps/festival/festival-e2e/src/integration/dashboard/screening-invitations-2688.spec.ts]: Super Admin.. will update comment.

File: env/env.blockframes-mano.ts (hunk 2 above)
Comment (xmano) [reply env/env.blockframes-mano.ts:145]: yes.

**Pull request merged**